src/VecSim/utils/vec_utils.h

```diff
@@ -12,8 +13,44 @@ struct CompareByFirst {
     }
 };
 
+struct VecSimCommonStrings {
```

Comment:
Do you think that we need a class for this? Shouldn't we just use `#define`?

Reply:
I prefer to have a dedicated container for this and avoid `#define`. I think RediSearch also defines its own strings. I don't want to have collisions here.